reactive-review/AppFlow.adoc

```diff
@@ -0,0 +1,12 @@
+:gls_prefix:
+
+// Do not modify section headers, as they are used by flamel and the translation process.
+= pass:a,n[{gls_res_outcomes}]
+
+You should be able to create Quarkus Conference application, by using Reactive Architecture and Principles.
+
+== Proposed App Flow
+
+* Create `session mS` a `Reactive` application using Quarkus.
+* Create an `asynchronous REST` endpoint by using RESTeasy Reactive (Attaching an image for the same).
+* Apply `event driven` principles on Quarkus application with `Reactive Messaging`.
```

Comment:
For the eda part, we need at least 2 services to communicate through Kafka and send events/data (I prefer events than simple data).
One of the services must use an `@Outgoing` channel, and the other service must use an `@Incoming` channel to process the message.

That's the minimum we must cover, if we want to go bananas with the apps, we can even set some logic to discard events, or make a chain process.

Reply:
Sounds good.

So, if we are using _Session_ and _Speaker_ services for this lab, we can make Session service as @Outgoing channel and speaker service as @Incoming channel.

Reply:
I think that using the _Session/Speaker_ app is going to be tricky.. mainly because the business logic that we want to model is not easy. To what event do we want to react, and to do what?

Reply:
Let me start with the initial steps and then we can figure out for the event part later.